content/influxdb/cloud/notebooks/downsample.md

```diff
@@ -31,14 +31,9 @@ Learn how to create a notebook that downsamples data—walk through the followin
   - In the **Choose a bucket** dropdown list, select **Website Monitoring Bucket**.
   - Select the **response_time** field.
   - Select the **mean** function from the aggregate selector in the upper-right.
-3. Add a **Flux Script** cell with the following script to bring in data from the previous cell and downsample it:
-  ```sh
-  __PREVIOUS_RESULT__
-    |> aggregateWindow(fn: mean, every: 1h)
-  ```
-  {{% note %}}
-   If the `every` duration is longer the the total time range queried, aggregateWindow will only return one value.
-  {{% /note %}}
+3. Add a **Downsample** cell with the following script to bring in data from the previous cell and downsample it.
+  - Select an aggregate function from the **Apply aggregate** dropdown menu.
+  - Enter a window period. 
```

Comment:
There isn't a script following this step, so I assume the data pipeline is inferred and they don't need `__PREVIOUS_RESULT__`.

```suggestion
3. Add a **Downsample** cell to bring in data from the previous cell and downsample it.
  - Select an aggregate function from the **Apply aggregate** dropdown menu.
  - Enter a window period. 
```